bionic/flow.py

```diff
@@ -116,7 +116,7 @@ def define_entity(self, entity_def):
                 `declare` / `assign` the entity values.
                 """
                 raise AttributeValidationError(oneline(message))
-            if not entity_def.can_memoize:
+            if not entity_def.should_memoize:
```

Comment:
I think that's okay since this is an internal entity. We want to be explicit about them. I can change this to be `if entity_def.should_memoize is None or not entity_def.should_memoize`. Wdyt?

Reply:
I think that's reasonable, as long as the error message is consistent with the logic. (Right now it only says `set @memoize to False`.)